drivers/cache/Kconfig.aspeed

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2021 ASPEED Technology Inc.
+
+config CACHE_ASPEED
+        bool "ASPEED external cache driver"
+        depends on SOC_SERIES_AST10X0
+        select CACHE_HAS_DRIVER
+        select SYSCON
+        select SYSCON_GENERIC
```

Comment:
uhm, this should `depend` on syscon, and syscon should be enabled by defconfig (or some other Kconfig file) instead.

Reply:
Thank you for your comment.  I've update the commits.